components/Settings.js

```diff
@@ -286,16 +262,171 @@ const MenuButton = React.memo(({ name, select, selected }) => {
           background-color: ${COLORS.SECONDARY};
         }
       `}
-      </style>
-    </div>
+    </style>
+  </div>
+))
+
+const Presets = React.memo(
+  React.forwardRef(
```

Comment:
@raboid you don't need to use `forwardRef` unless the wants to pass the ref in the `ref` prop (https://reactjs.org/docs/forwarding-refs.html) You can just remove it in these case since you are using the `contentRef` prop instead

Reply:
Oh interesting I had thought it was needed in general to pass a ref created in the parent to a child. Thanks!